### DISCUSSION TITLE
能不能选择屏蔽一下番号前面的数字

### DISCUSSION THREAD

**Discussion by zjj86452284** [metatube-community/jellyfin-plugin-metatube#298] [General]
我是把avbase作为默认源，而部分A.V. 用avbase会刮削avbase上面的mgstage的数据，往往都带有一些类似什么002HODV-等字样，这得要手动删除，一旦片源多了，就好难受了。其他刮削源，要么就是简介很简短，甚至是没有简介，要么就是没有演员信息。
![QQ截图20240303115130](https://github.com/metatube-community/jellyfin-plugin-metatube/assets/106129075/4623e0c3-e3f8-47bd-b0a9-9258260d5058)

**Comment by callerin** [comment]
同感，目前只能自己写了个Python脚本批量修改nfo文件了

**Reply by zjj86452284** [reply]
我刮削没创建NFO文件，直接就是靠metatube刮削的。NFO文件和封面那些，虽然容量相对比较少，但是片源一多，挤一挤也能挤出一部片子的容量了
